LeoProjectGaming/src/main/java/com/work/LeoProjectGaming/service/GamingService.java

```diff
@@ -0,0 +1,41 @@
+package com.work.LeoProjectGaming.service;
+
+import com.work.LeoProjectGaming.entity.Bet;
+import com.work.LeoProjectGaming.repository.BettingHistoryRepository;
+import org.example.LeoProjectKafkaDTOS.BettingTransferDTO;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+import java.util.Date;
+import java.util.Random;
+
+import static com.work.LeoProjectGaming.util.Constants.BETTING_TRANSFER_TOPIC;
+
+@Service
+public class GamingService {
+
+    public static final Random winOrLose = new Random();
```

Comment:
Making Random static final variable here seems tricky. I would do that only for constants.